Source/WebCore/platform/win/WebCoreBundleWin.cpp

```diff
@@ -61,6 +66,92 @@ CFBundleRef webKitBundle()
     return bundle.get().get();
 }
 
-} // namespace WebCore
+String webKitBundlePath()
+{
+    URL bundleURL = adoptCF(CFBundleCopyBundleURL(webKitBundle())).get();
+    return bundleURL.fileSystemPath();
+}
 
-#endif
+String webKitBundlePath(StringView path)
+{
+    auto pathString = path.toStringWithoutCopying();
+    auto directory = FileSystem::parentPath(pathString);
+    auto fileName = FileSystem::pathFileName(pathString);
+
+    auto parts = fileName.split('.');
```

Comment:
We don’t need to do a split, since we are really just trying to shave off the last period-separated piece. We should use `reverseFind` and then we can use `substring`, `left`, and `right`.

If we do want to stick with `split`, then it could be more efficient if we use the `split` from `StringView` rather than the one from `String`.